data/json/professions.json

```diff
@@ -1716,16 +1717,16 @@
     "type": "profession",
     "id": "crossbow_hunter",
     "name": "Crossbow Hunter",
-    "description": "Ever since you were a child you loved hunting, and crossbow hunting was always your favorite.  Why, if the world ended, there's nothing you'd want at your side more than your trusty crossbow.  So, when it did, you made sure to bring it along.",
-    "points": 2,
+    "description": "Ever since you were a child you loved hunting, and crossbow hunting was always your favorite.  Why, if the world ended, there's nothing you'd want at your side more than your trusty crossbow.  So, when it did, you made sure to bring it along.  But most of your bolts are lost during the escape",
```

Comment:
What do you think of a slight rephrasing,
>"Ever since you were a child you loved hunting, and crossbow hunting was always your favorite. Why, if the world ended, there's nothing you'd want at your side more than your trusty crossbow. So, when it did, you made sure to bring it along, though most of your bolts were lost during the escape."

Reply:
Compound crossbow is too powerful, so I reduce the number of bolts to balance it. This rephrasing is to makes it not abrupt

Reply:
Thanks for your advice. I misunderstood at the beginning.